### DISCUSSION TITLE
Why wouldn't `curl` work, but `curl -4` would?

### DISCUSSION THREAD

**Discussion by Lippiece** [nextcloud/all-in-one#4832] [Questions] [answered]
I'm using the [rootless podman quadlet setup](https://github.com/nextcloud/all-in-one/discussions/3487).
For some reason, curl fails when run inside the container, but curl -4 works normally. As such:
NOTICE: PHP message: Could not get digest of container nextcloud/aio-domaincheck:latest cURL error 7: Failed to connect to auth.docker.io port 443 after 4003 ms: Couldn't connect to server (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://auth.docker.io/token?service=registry.docker.io&scope=repository:nextcloud/aio-domaincheck:pull
I suspect this is due to some ipv6 misconfiguration. Can someone hint me where to look?

```
  [Unit]
  Description=Nextcloud AIO Master Container
  Documentation=https://github.com/nextcloud/all-in-one/blob/main/docker-rootless.md
  After=local-fs.target
  Requires=podman.socket
  
  [Container]
  ContainerName=nextcloud-aio-mastercontainer
  Image=docker.io/nextcloud/all-in-one:latest
  Network=bridge
  SecurityLabelDisable=true
  AutoUpdate=registry
  
  PublishPort=11001:8080
  PublishPort=4001:8000
  
  Volume=nextcloud_aio_mastercontainer:/mnt/docker-aio-config
  Volume=/run/user/1000/podman/podman.sock:/var/run/docker.sock
  
  Environment=APACHE_PORT=11000
  Environment=APACHE_IP_BINDING=0.0.0.0
  Environment=WATCHTOWER_DOCKER_SOCKET_PATH=/run/user/1000/podman/podman.sock
  Environment=SKIP_DOMAIN_VALIDATION=true
  
  Sysctl=net.ipv6.conf.default.disable_ipv6=1 net.ipv6.conf.all.disable_ipv6=1 net.ipv6.conf.all.disable_ipv6=1
  
  [Install]
  WantedBy=multi-user.target default.target
```

**Comment by szaimen** [comment, marked as answer]
Hi, as stated in the readme, [Podman is not supported](https://github.com/nextcloud/all-in-one?tab=readme-ov-file#can-i-run-this-with-podman-instead-of-docker). Thus, please keep the discussion in this thread: https://github.com/nextcloud/all-in-one/discussions/3487. Thanks!
